packages/fxa-settings/src/components/Settings/ButtonIcon/index.stories.tsx

```diff
@@ -3,17 +3,29 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { ButtonIconTrash, ButtonIconReload } from '.';
+import { Meta } from '@storybook/react';
+import { withLocalization } from '../../../../.storybook/decorators';
 
-storiesOf('Components/Settings/ButtonIcon', module)
-  .add('ButtonIconTrash', () => (
-    <div className="p-10 max-w-lg">
-      <ButtonIconTrash title="Remove email" />
-    </div>
-  ))
-  .add('ButtonIconReload', () => (
-    <div className="p-10 max-w-lg">
-      <ButtonIconReload title="Reload email" />
-    </div>
-  ));
+export default {
+  title: 'Components/Settings/ButtonIcon',
+  component: ButtonIconTrash, ButtonIconReload,
```

Comment:
Q: What does this line do?
I'm a bit confused by the comma here between components since it seems like we're setting `component: ButtonIconTrash` but then `ButtonIconReload` isn't associated w/ `component` and is being passed as a separate`ButtonIconReload: ButtonIconReload` prop.

A quick grep shows that this might be the only comma-separated `component` in the stories:
```sh
git grep -n "component: " packages/**/*.stories.* | grep ", "

# packages/fxa-settings/src/components/Settings/ButtonIcon/index.stories.tsx:12:  component: ButtonIconTrash, ButtonIconReload,
```

I can't easily find anything on the Storybook docs about passing an array of `components`, but I did find a mention of `subcomponents` at https://storybook.js.org/docs/react/writing-docs/docs-page#subcomponents-parameter